### PR TITLE
Update demo apps' configs to address dependabot alerts

### PR DESCRIPTION
**Pull request by bangtoven**
### _Summary_

addressing dependabot alerts
image

### _How did you test your changes?_

manually
Screenshot 2023-01-11 at 5 18 22 PM

### CODE REVIEW

File: examples/rainbowkit-demo/package.json [hunk 1]
```diff
@@ -10,17 +10,20 @@
...
-    "react-scripts": "5.0.1",
     "util": "^0.12.4",
     "wagmi": "^0.9.6"
   },
+  "devDependencies": {
+    "react-app-rewired": "^2.2.1"
+  },
+  "overrides": {
+    "react-scripts": {
+      "nth-check": "^2.0.1"
```
Comment (bangtoven) [inline examples/rainbowkit-demo/package.json:24]: to address dependabot alerts 
image
https://github.com/facebook/create-react-app/issues/12132#issuecomment-1130249584

**Review by taycaldwell — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for taycaldwell @ 2023-01-12 02:31:58 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by taycaldwell — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for taycaldwell @ 2023-01-12 02:32:22 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by taycaldwell — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for taycaldwell @ 2023-01-12 02:33:41 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by taycaldwell — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for taycaldwell @ 2023-01-12 02:37:12 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by taycaldwell — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for taycaldwell @ 2023-01-12 02:40:11 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Comment by taycaldwell** [comment]
Ugh. Need to wait for Heimdall cache to clear or something.

**Review by taycaldwell — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for taycaldwell @ 2023-01-12 02:43:46 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by taycaldwell — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for taycaldwell @ 2023-01-12 02:45:09 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by taycaldwell — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for taycaldwell @ 2023-01-12 02:46:07 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Comment by taycaldwell** [comment]
Layoff Heimdall pls.

**Review by taycaldwell — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for taycaldwell @ 2023-01-12 02:50:06 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by taycaldwell — APPROVED** [review]

**Review by vishnumad — APPROVED** [review]

**Review by erin-at-work — APPROVED** [review]

**Review by vishnumad — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for vishnumad @ 2023-01-12 18:34:36 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Pull request merged**